package.json

```diff
@@ -62,7 +62,7 @@
     "datatables.net-scroller": "1.4.2",
     "datatables.net-scroller-jqui": "1.4.2",
     "dropzone": "4.3.0",
-    "dwv": "0.21.0",
+    "dwv": "^0.24.0",
```

Comment:
Odd to see this. Why was this changed?

Reply:
I too am curious as I did not work on that file, I made a minor modification to 
`interface/main/tabs/menu/menus/patient_menus/standard.json`